### ISSUE TITLE
It is not showing

### ISSUE THREAD

**Issue by thejuanvisu** [darkjuanjo/Persona_5_Calendar#2]
Apparently it is working correctly and shows on the ubersicht menu but it doesnt appear on my desktop. ¿Any idea of what can i do?

**Comment by darkjuanjo** [comment]
hmm is it enabled?

<img width="206" alt="Screen Shot 2021-11-30 at 3 35 11 PM" src="https://user-images.githubusercontent.com/25261067/144145038-917f7fc4-b682-4a06-b04e-3e8ba77defa4.png">
<img width="401" alt="Screen Shot 2021-11-30 at 3 34 15 PM" src="https://user-images.githubusercontent.com/25261067/144145040-a62c7bd7-fe0d-481c-bf14-4f449e301f18.png">

**Comment by thejuanvisu** [comment]
Yep, i tried that before coming here, i dont know why it does not work, it looks like other widgets work without problems
![imagen](https://user-images.githubusercontent.com/61268500/144679717-4b7bd455-0664-4dd7-b98c-acb8662cf10a.png)

**Comment by darkjuanjo** [comment]
hmm are there any issues on the debug console? 
maybe your digital clock somehow overlaps the calendar?

**Comment by keijello** [comment]
Does anyone have any updates or fixes on this? The same thing happened to me, and it's the only widget I currently have. 
I tried editing the API call URL and retrying different API keys, but neither change it. Currently it just looks like this: 
<img width="298" alt="Screenshot 2024-01-04 at 5 47 01 PM" src="https://github.com/darkjuanjo/Persona_5_Calendar/assets/155691429/ac4d8700-ea17-45d4-ad8e-e2790e1a3751">

**Comment by darkjuanjo** [comment]
So nothing is working? Can you do console print the output of the CPU and the API call to see what we are getting? What OS are you running? Maybe the commands need to be updated?> Does anyone have any updates or fixes on this? The same thing happened to me, and it's the only widget I currently have. 
> 
> I tried editing the API call URL and retrying different API keys, but neither change it. Currently it just looks like this: 
> 
> <img width="298" alt="Screenshot 2024-01-04 at 5 47 01 PM" src="https://github.com/darkjuanjo/Persona_5_Calendar/assets/155691429/ac4d8700-ea17-45d4-ad8e-e2790e1a3751">
> 
>

**Comment by keijello** [comment]
<img width="436" alt="Screenshot 2024-01-07 at 6 10 46 PM" src="https://github.com/darkjuanjo/Persona_5_Calendar/assets/155691429/d589be6f-7629-41fd-bbab-ca63ff4b2f42">

This is currently what the Calendar object looks like in network. I'll be honest, I really am unfamiliar with this, so I'm not sure if I should be changing something to have the dates change. However, the weather does seem to be working. 


OS: I'm currently running Sonoma 14.1.1 
Current API Call Result in Network (weather):
<img width="519" alt="Screenshot 2024-01-07 at 6 05 51 PM" src="https://github.com/darkjuanjo/Persona_5_Calendar/assets/155691429/eeb9d69a-842f-4f68-9bce-914bcc4543f4">
Current Location String: (In VSC and folder info on mac) 
<img width="232" alt="Screenshot 2024-01-07 at 6 09 25 PM" src="https://github.com/darkjuanjo/Persona_5_Calendar/assets/155691429/ac3960eb-0bc0-46c5-b9e8-fcfad5aec7dc">

Also, when I look in network at any of the pngs, they all load like this. But they load just fine in my files. I tried changing the folder string just in case it didn't have access, but it still seems to show up this way even with full disk access (shows as either ? or as an html (no, the files are still pngs))
<img width="443" alt="Screenshot 2024-01-07 at 6 13 00 PM" src="https://github.com/darkjuanjo/Persona_5_Calendar/assets/155691429/83ff5ed6-25a9-49fd-a752-c26c95533e44">
<img width="436" alt="Screenshot 2024-01-07 at 6 20 11 PM" src="https://github.com/darkjuanjo/Persona_5_Calendar/assets/155691429/ec1d9e93-25ff-418a-b967-ff32655dc952">

Thank you! No stress honestly, but I just assume I don't really know the answers. If it's too messy of a fix then no worries.

**Comment by darkjuanjo** [comment]
@keijello maybe the issue is with the location. Can you change the location to this: Folder: "Persona_5_Calendar/Calendar_Assets/",

**Comment by keijello** [comment]
Still the same result :/

**Comment by darkjuanjo** [comment]
Can you run this command in the terminal: date +"%m %d %A %H"

**Comment by keijello** [comment]
Yes, it results in plainly the date: 01 07 Sunday 20

**Comment by darkjuanjo** [comment]
Hmm, not sure what is wrong. The file location seems correct. Is this in the widget folder?

**Comment by keijello** [comment]
It does the same thing whether I direct it to a different folder or inside the widgets (with the correct string)
I'll keep trying to troubleshoot, but thank you for the help and replies tho!

**Comment by ForestEden** [comment]
Did any of you ever figure this out? My weather also shows as a question marl

**Comment by darkjuanjo** [comment]
> Did any of you ever figure this out? My weather also shows as a question marl

So it doesn't show anything? Can you share the console error?
